### ISSUE TITLE
Custom rule for type check doesn't work

### ISSUE THREAD

**Issue by BrunaBoaventura** [looker-open-source/look-at-me-sideways#23]
I currently have 2 custom rules in my manifest file. One works which is the production connection check:
```
# rule: prod_connection {
#  description: "Models must not use sandbox to deploy"
#  match: "$.model.*"
#  expr_rule: ( $boolean($match "sandbox^" ::match:connection) ) ;;
# }
```
But the second one doesn't work. It is supposed to check if the type is available, but I tried writing it as the example here:
```
# rule: types_required {
#	match: "$.model.*.view.*['dimension','measure'].*"
#	description: "Fields must specify a type"
#	expr_rule: ( !== ::match:type undefined ) ;;
# }
```
But this doesn't work. To make sure that the lookml code isn't the issue, I tried changing the rule slightly since I'm sure I have dimensions and measures that are not type "string":
```
# rule: fields_must_have_type {
#  description: "All dimensions and measures must have a type defined"
#  match: "$.model.*.view.*['dimension','measure'].*"
#  expr_rule: ( !== ::match:type "string" ) ;;
# }
```
Do you have any insights on why this rule is not working?
Thank you

**Comment by fabio-looker** [comment]
(By coincidence?) This is exactly the rule I used in the most recent test case:

![image](https://user-images.githubusercontent.com/23034640/69567984-4bca4880-0f80-11ea-8666-c6916d493623.png)

When I run that test case, I get the following:
![image](https://user-images.githubusercontent.com/23034640/69572694-5093fa00-0f8a-11ea-97e0-9cd1ae6e6ac6.png)

Just to be sure this is not a broader issue that is not caught by the unit test, I also ran LAMS directly in the test directory and got the expected issues.md output:
![image](https://user-images.githubusercontent.com/23034640/69572996-dadc5e00-0f8a-11ea-92e5-99a079f24fc4.png)

Did you install LAMS recently? (i.e. post v1?)

As a side note, I think you have a regex issue in your first rule. Since `^` is string start, you probably want either `^sandbox` or `sandbox$`
